server/odc-migrate/src/main/resources/migrate/common/V_4_2_4_1__alter_notification_schema.sql

```diff
@@ -0,0 +1,108 @@
+  `creator_id` bigint(20) NOT NULL comment 'creator user id, references iam_user.id',
+  `organization_id` bigint(20) NOT NULL comment 'organization id, references iam_organization.id',
+  `title_template` text DEFAULT NULL comment 'notification message title template, which could contains variables',
+  `content_template` text DEFAULT NULL comment 'notification message content template, which could contains variables',
+  `match_expression` varchar(2048) NOT NULL comment 'indicate if a event matches the expression',
+  `to_users` varchar(2048) DEFAULT NULL comment 'odc users who will receive this message',
+  `cc_users` varchar(2048) DEFAULT NULL comment 'odc users who will receive this message by copy',
+  `is_enabled` tinyint(1) NOT NULL DEFAULT '1' COMMENT 'Flag bit, mark whether the triggered rule is enabled',
+  `project_id` bigint(20) NOT NULL COMMENT 'project id, references collaboration_project.id',
+  `policy_metadata_id` bigint(20) NOT NULL COMMENT 'notification policy metadata id, reference to notification_policy_metadata.id',
+  CONSTRAINT `pk_notification_policy_id` PRIMARY KEY (`id`),
+  KEY `notification_policy_project_id` (`project_id`)
+) COMMENT = 'notification policy';
+
+CREATE TABLE IF NOT EXISTS `notification_policy_metadata`(
+  `id` bigint(20) NOT NULL AUTO_INCREMENT,
+  `create_time` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  `update_time` datetime DEFAULT CURRENT_TIMESTAMP ON update CURRENT_TIMESTAMP,
+  `event_category` varchar(128) NOT NULL comment 'notification event category, such as TASK',
+  `event_name` varchar(128) NOT NULL comment 'notification event name',
+  `match_expression` varchar(2048) comment 'indicate if a event matches the expression',
+  CONSTRAINT `pk_notification_policy_metadata_id` PRIMARY KEY (`id`),
+  CONSTRAINT `uk_policy_metadata_event_category_name` UNIQUE KEY (`event_category`, `event_name`)
+) comment = 'notification policy metadata';
+
+DROP TABLE IF EXISTS `notification_event`;
+CREATE TABLE IF NOT EXISTS `notification_event`(
+  `id` bigint(20) NOT NULL AUTO_INCREMENT,
+  `create_time` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  `update_time` datetime DEFAULT CURRENT_TIMESTAMP ON update CURRENT_TIMESTAMP,
+  `creator_id` bigint(20) NOT NULL comment 'creator user id, references iam_user.id',
+  `organization_id` bigint(20) NOT NULL comment 'organization id, references iam_organization.id',
+  `project_id` bigint(20) NOT NULL comment 'project id, references collaboration_project.id',
+  `trigger_time` datetime NOT NULL comment 'time when the event triggered',
+  `status` varchar(128) NOT NULL comment 'status enum, may CREATED, THROWN, CONVERTED, etc.',
+  CONSTRAINT `pk_notification_event_id` PRIMARY KEY (`id`),
+  KEY `notification_event_status_time`(`status`, `trigger_time`)
+) comment = 'notification events';
+
+DROP TABLE IF EXISTS `notification_message`;
+CREATE TABLE IF NOT EXISTS`notification_message` (
```

Comment:
add a white space between 'EXISTS' and '`notification_message `'?

Reply:
done